setup-utils/install-mxnet-osx-python.sh

```diff
@@ -8,32 +8,54 @@
 
 #set -ex
 
+export MXNET_GITPATH="https://github.com/dmlc/mxnet.git"
+if [ x"${MXNET_TAG}" = "x" ];
+then
+	#
+	# TODO: Change this to latest tag 
+	#       to avoid updating this value for every release
+	#
+	export MXNET_TAG="v0.10.0"
+fi
+
 export TARIKH=`/bin/date +%Y-%m-%d-%H:%M:%S`
-export MXNET_HOME="$HOME/mxnet"
+if [ x"${MXNET_HOME}" = "x" ];
+then
+	export MXNET_HOME="$HOME/mxnet"
+fi
 export MXNET_HOME_OLD="$HOME/mxnet_${TARIKH}"
```

Comment:
An old directory does get created by the lines: 
if [ -d ${MXNET_HOME} ]; then
                mv ${MXNET_HOME} ${MXNET_HOME_OLD}